### PR TITLE
[Compatibility] Fixing version comparision bug in python3

### PR DESCRIPTION
**Pull request by jostep**

### CODE REVIEW

**Comment by windreamer** [comment]
maybe not necessary?
cf: https://github.com/douban/pymesos/blob/master/pymesos/scheduler.py#L30

**Comment by jostep** [comment]
This is the error msg without the fix. 
I am using pymesos 0.2.9.


giorgio@giorgio:~/Github/tfmesos/examples|master⚡ 
⇒  python3 plus.py giorgio:5050
2016-12-16 00:18:11,624 [INFO] [tfmesos.scheduler] Tensorflow cluster registered. ( http://giorgio:5050/#/frameworks/289dee14-98aa-4dbe-bdbd-0637d3f671aa-0008 )
INFO:tfmesos.scheduler:Tensorflow cluster registered. ( http://giorgio:5050/#/frameworks/289dee14-98aa-4dbe-bdbd-0637d3f671aa-0008 )
ERROR:pymesos.process:Failed to process event
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/pymesos/process.py", line 143, in read
    self._callback.process_event(event)
  File "/usr/local/lib/python3.5/dist-packages/pymesos/process.py", line 221, in process_event
    self.on_event(event)
  File "/usr/local/lib/python3.5/dist-packages/pymesos/scheduler.py", line 448, in on_event
    func(event)
  File "/usr/local/lib/python3.5/dist-packages/pymesos/scheduler.py", line 392, in on_subscribed
    self._dict_cls(master_info)
  File "/usr/local/lib/python3.5/dist-packages/tfmesos-0.0.1-py3.5.egg/tfmesos/scheduler.py", line 341, in registered
    'MESOS' if driver.version >= (1, 0, 0) else 'DOCKER'
TypeError: unorderable types: str() >= tuple()
ERROR:pymesos.process:Thread abort:
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/pymesos/process.py", line 303, in _run
    if not conn.read():
  File "/usr/local/lib/python3.5/dist-packages/pymesos/process.py", line 143, in read
    self._callback.process_event(event)
  File "/usr/local/lib/python3.5/dist-packages/pymesos/process.py", line 221, in process_event
    self.on_event(event)
  File "/usr/local/lib/python3.5/dist-packages/pymesos/scheduler.py", line 448, in on_event
    func(event)
  File "/usr/local/lib/python3.5/dist-packages/pymesos/scheduler.py", line 392, in on_subscribed
    self._dict_cls(master_info)
  File "/usr/local/lib/python3.5/dist-packages/tfmesos-0.0.1-py3.5.egg/tfmesos/scheduler.py", line 341, in registered
    'MESOS' if driver.version >= (1, 0, 0) else 'DOCKER'
TypeError: unorderable types: str() >= tuple()
2016-12-16 00:18:11,722 [DEBUG] [tfmesos.scheduler] exit
DEBUG:tfmesos.scheduler:exit
Traceback (most recent call last):
  File "plus.py", line 38, in <module>
    main(sys.argv)
  File "plus.py", line 22, in main
    with cluster(jobs_def, master=mesos_master, quiet=False) as targets:
  File "/usr/lib/python3.5/contextlib.py", line 59, in __enter__
    return next(self.gen)
  File "/usr/local/lib/python3.5/dist-packages/tfmesos-0.0.1-py3.5.egg/tfmesos/__init__.py", line 21, in cluster
  File "/usr/local/lib/python3.5/dist-packages/tfmesos-0.0.1-py3.5.egg/tfmesos/scheduler.py", line 312, in start
  File "/usr/local/lib/python3.5/dist-packages/tfmesos-0.0.1-py3.5.egg/tfmesos/scheduler.py", line 295, in readable
  File "/usr/local/lib/python3.5/dist-packages/pymesos/process.py", line 25, in _handle_sigint
    reraise(*exc_info)
  File "/usr/lib/python3/dist-packages/six.py", line 686, in reraise
    raise value
  File "/usr/local/lib/python3.5/dist-packages/pymesos/process.py", line 303, in _run
    if not conn.read():
  File "/usr/local/lib/python3.5/dist-packages/pymesos/process.py", line 143, in read
    self._callback.process_event(event)
  File "/usr/local/lib/python3.5/dist-packages/pymesos/process.py", line 221, in process_event
    self.on_event(event)
  File "/usr/local/lib/python3.5/dist-packages/pymesos/scheduler.py", line 448, in on_event
    func(event)
  File "/usr/local/lib/python3.5/dist-packages/pymesos/scheduler.py", line 392, in on_subscribed
    self._dict_cls(master_info)
  File "/usr/local/lib/python3.5/dist-packages/tfmesos-0.0.1-py3.5.egg/tfmesos/scheduler.py", line 341, in registered
TypeError: unorderable types: str() >= tuple()
giorgio@giorgio:~/Github/tfmesos/examples|master⚡

**Comment by windreamer** [comment]
Ah, I see

File: tfmesos/scheduler.py [hunk 1]
```diff
@@ -338,7 +338,7 @@ def registered(self, driver, framework_id, master_info):
 
         if self.containerizer_type is None:
             self.containerizer_type = (
-                'MESOS' if driver.version >= (1, 0, 0) else 'DOCKER'
+                'MESOS' if [int(x) for x in driver.version.split(".")] >= (1, 0, 0) else 'DOCKER'
```
Comment (windreamer) [inline tfmesos/scheduler.py]: Better this way:
```
version = tuple(int(x) for x in driver.version.split("."))
...
'MESOS' if version >= (1, 0, 0) else 'DOCKER'
```

**Comment by windreamer** [comment]
OK, let's merge this, and I will refactor it later

**Pull request merged**